sdk-src/src/main/java/com/hpe/adm/nga/sdk/Octane.java

```diff
@@ -88,6 +89,10 @@ public EntityList entityList(String entityName) {
         return OctaneClassFactory.getSystemParamImplementation().getEntityList(octaneHttpClient, getBaseDomainFormat(), entityName);
     }
 
+    public <T extends TypedEntityList> T entityList(Class<T> entityListClass) {
```

Comment:
so it's almost the same as generating the Octane object itself, i see, nevermind then :(